hardhat/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+name: lilypad-chain
+
+services:
+  #lilypad-node:
+  #  build:
+  #    context: infra/ava
+  #    dockerfile: Dockerfile
+  #  entrypoint: tail -f /dev/null
+  #  ports:
+  #    - 9650:9650
+  #    - 9652:9652
+  #    - 9654:9654
+  #    - 9656:9656
+  #    - 9658:9658
+  
+  lilypad-geth:
+    build:
+      context: infra/geth
```

Comment:
So `ava` runs a `geth subnet`? Is this the way the testnet and mainnet chains work too? My question is, when we deploy there, is there a subnet that we deploy to or could you help me wrap my mental model around it?

Reply:
no, i added both geth and avalanche testnets. avalanche is the preferred route, but there was existing work with geth that i cleaned up.

```bash
cd hardhat
./scripts/setup-testnet.sh
./scripts/start-testnet.sh
npx hardhat deploy --network lilypad
```